src/olympia/scanners/migrations/0034_delete_version_flags_again.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.12 on 2020-05-19 14:38
+
+from django.db import migrations
+
+
+def delete_version_scanner_flags(apps, schema_editor):
+    # We have to delete all the existing entries (again) because we did not
+    # have the logic to clear the flags after a review, so all versions were
+    # kept in the queue. The feature is kinda new so it is fine to delete
+    # existing entries.
+    # See: https://github.com/mozilla/addons-server/issues/14375
```

Comment:
you could work it out based on `Version.needs_human_review`?  But if @wagnerand doesn't care then 🤷‍♂️

Reply:
It's fine, it is an experimental queue anyway..